src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/HearingDocumentController.java

```diff
@@ -84,6 +88,21 @@ public ResponseEntity<AboutToStartOrSubmitCallbackResponse> generateHearingDocum
         }
 
         List<String> warnings = validateHearingService.validateHearingWarnings(caseDetails);
+
+
+        // NOTE TO SELF, TEST BOTH PAPER AND DIGITAL JOURNEYS
```

Comment:
did you forget removing this? as a tip, I'd use the flag `TODO:` on comments like this as IntelliJ marks them and reminds you when you are about to push with a warning.